sphinx/source/docs/dev_guide/building.rst

```diff
@@ -119,12 +119,12 @@ Building BokehJS with SBT
         The ``sbt`` build system is experimental and not integrated with
         ``setup.py``, so it should be used with caution.
 
-As an alternative to Grunt, you can use `SBT`_ to build BokehJS. To start,
+As an alternative to Gulp, you can use `SBT`_ to build BokehJS. To start,
```

Comment:
Is there a reason to support it? And even if it still works, is there a reason to document this experimental feature? It add to the clutter IMO.